src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html

```diff
@@ -1,127 +1,193 @@
+                            <span>({{ exam.startDate | artemisDate: 'time' }})</span>
+                        </div>
                     }
-                    <button id="start-exam" [disabled]="!startButtonEnabled || waitingForExamStart" type="submit" (click)="startExam()" class="btn btn-primary">
-                        <span jhiTranslate="artemisApp.exam.startExam"></span>
-                    </button>
-                </ng-container>
-            } @else {
+                </div>
+            }
+        </div>
+    } @else {
+        <div class="container d-flex flex-column align-items-center justify-content-center">
+            <div>
+                <p class="font-weight-bold text-align-center">
+                    <span jhiTranslate="artemisApp.examParticipation.submitFinalExam"></span>
+                </p>
+            </div>
+            <div class="form-check mt-1 ps-0">
+                <input
+                    [(ngModel)]="confirmed"
+                    type="checkbox"
+                    id="confirmBox"
+                    (click)="updateConfirmation()"
+                    class="form-check-input"
+                    [class.ms-0]="!this.exam.confirmationEndText"
+                    [required]="inserted"
+                />
+                <label for="confirmBox" id="formatted-confirmation-text" class="form-check-label" [innerHTML]="formattedConfirmationText"></label>
+            </div>
+            <div style="text-align: center">
+                <div class="login-form">
+                    <div class="row justify-content-center">
+                        <div class="col-md-6">
+                            <label class="font-weight-bold" for="fullname" jhiTranslate="{{ 'artemisApp.exam.endConsentText' }}"></label>
+                        </div>
+                    </div>
+                    <div class="row justify-content-center" style="padding-bottom: 0; margin-bottom: 0">
+                        <div class="form-group pb-0 mb-0" style="width: unset">
+                            <input
+                                size="50"
+                                type="text"
+                                class="form-control"
+                                name="fullname"
+                                id="fullname"
+                                [placeholder]="'artemisApp.examParticipation.namePlaceholder' | artemisTranslate"
+                                [(ngModel)]="enteredName"
+                                [ngModelOptions]="{ updateOn: 'change' }"
+                            />
+                        </div>
+                    </div>
+                    <div class="row justify-content-center">
+                        <div id="your-name" class="col-md-6 fw-lighter">
+                            (<span jhiTranslate="artemisApp.examParticipation.nameDisplay" [translateValues]="{ fullName: accountName }"></span>)
+                        </div>
+                    </div>
```

Comment:
There is quite some duplicated code and some duplicated ids in this HTML file that even cause errors in the IDE - a better approach would be to use ngTemplates here

Due to the urgency this does not need to be adjusted - but could you add a comment to the code why these ids are duplicated if it was an active decision?